src/interpreter.js

```diff
@@ -264,37 +243,37 @@ const compileArguments = (args) => args.map(compileExpression).join()
 
 const compileSentenceSequence = (sentences) => {
   const compiledSentences = sentences.map(compileSentence)
-  if(compiledSentences.length && !compiledSentences[compiledSentences.length - 1].startsWith('return'))
-    compiledSentences[compiledSentences.length - 1] = 'return ' + compiledSentences[compiledSentences.length - 1]
+  if (compiledSentences.length && !compiledSentences[compiledSentences.length - 1].startsWith('return'))
+    compiledSentences[compiledSentences.length - 1] = `return ${compiledSentences[compiledSentences.length - 1]}`
   return compiledSentences.join('')
 }
 
+const findByType = (content, type) => content.filter(c => c.nodeType === type)
+
 //TODO: Imports
-const compileFile = ({content}) => {
-  const imports = content.filter(c => c.nodeType === 'Import')
-  const programs = content.filter(c => c.nodeType === 'Program')
-  const tests = content.filter(c => c.nodeType === 'Test')
-  const elements = content.filter(c => c.nodeType !== 'Test' && c.nodeType !== 'Program' && c.nodeType != 'Import')
-
-  if(!programs.length && !tests.length) {
-    log.error(`There is no program or tests to run!`)
+const compileFile = ({ content }) => {
+  const imports = findByType(content, 'Import')
+  const programs = findByType(content, 'Program')
+  const tests = findByType('Test')
```

Comment:
But yes.. in xtext version imports at least are separated from the elements (like program, tests)